src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  margin-top: 0;
+  padding-top: 20px;
+  font-family: "Avenir", Arial, sans-serif;
+  font-weight: 300;
+}
+
+input {
+  font: inherit;
+}
+
+.searchForm {
+  position: relative;
+}
+
+.searchField {
+  display: block;
+  box-sizing: border-box;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  outline: none;
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.bigField {
+  height: 70px;
+  padding-left: 62px;
+  margin-bottom: 20px;
+  font-size: 16px;
+  line-height: 22px;
+}
```

Comment:
Move all common styles into searchField, and create modifiers like searchField--small searchField--big which will have non-common properties

Reply:
[checklist.md](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

[STYLES] - if you have two or more similar elements with portions of similar styles with different values - use one of the elements as the basic case, and override necessary styles for other cases. Explanation: The point is not in the names of the classes, the point is: when there are several similar elements, ex., 2 inputs, for one we can give a class input, for example, and for the second - input input--small. We write all the styles for .input, but for .input-small we write only those styles that differ in design, and we need this second input to look a little different. Element with class .input without extra classes should also look like a full-fledged styled element.

Is this another case?